### PR TITLE
Implement OutputFormat for block in Cli and ledger-tool bigtable

### PR DESCRIPTION
**Pull request by CriesofCarrots**
#### Problem
Not all cli commands have been updated to produce properly formatted output.
Also, consumers of `solana-ledger-tool bigtable block` would like to access the same json output as rpc/cli

#### Summary of Changes
- Implement OutputFormat for solana block
- Use in `solana-ledger-tool bigtable block`

Toward https://github.com/solana-labs/solana/issues/14509

### CODE REVIEW

**Comment by CriesofCarrots** [comment]
@mvines This changes the block-time and rewards output of non-json `solana-ledger-tool bigtable block` to match the cli. Any concerns?

Before:
```
Block Time: Some(1614055671)
Rewards: [Reward { pubkey: "DhbqhwX2J2mrCuFZxaUmCRvDfYihyrEgvHHbD6d8XWQe", lamports: 2161779828749, post_balance: 78921205051376, reward_type: Some(Voting) }, Reward { pubkey: "EpbWnY8D1XorvtwL2QVtQraMZmnEXhGJvdTyBWZZUCQa", lamports: 2077374324976, post_balance: 2613885840098, reward_type: Some(Voting) }]
```

After:
```
Block Time: 2021-02-22T21:47:51-07:00
Rewards:
  Address                                            Type        Amount           New Balance           Percent Change
  DhbqhwX2J2mrCuFZxaUmCRvDfYihyrEgvHHbD6d8XWQe      voting       ◎2161.779828749  ◎78921.205051376        2.816305389%
  EpbWnY8D1XorvtwL2QVtQraMZmnEXhGJvdTyBWZZUCQa      voting       ◎2077.374324976  ◎2613.885840098       387.200324024%
```

**Comment by mvines** [comment]
> @mvines This changes the block-time and rewards output of non-json `solana-ledger-tool bigtable block` to match the cli. Any concerns?

none, that's way better

File: cli-output/src/cli_output.rs [hunk 1]
```diff
@@ -1739,6 +1742,100 @@ impl fmt::Display for CliSignatureVerificationStatus {
...
+                "Address", "Type", "Amount", "New Balance", "Percent Change"
+            )?;
+            for reward in rewards {
+                let sign = if reward.lamports < 0 { "-" } else { "" };
+
+                total_rewards += reward.lamports;
+                writeln!(
+                    f,
+                    "  {:<44}  {:^15}  {:>15}  {}",
+                    reward.pubkey,
```
Comment (t-nelson) [inline cli-output/src/cli_output.rs:1790]: If we `.to_string()` here, the format width will be respected

**Review by t-nelson — APPROVED** [review]
LGTM!  Just one nit

File: cli-output/src/cli_output.rs (hunk 1 above)
Comment (CriesofCarrots) [reply cli-output/src/cli_output.rs:1790]: Sorry, I don't follow. `reward.pubkey` is already a String

Comment (t-nelson) [reply cli-output/src/cli_output.rs:1790]: Ah! I missed that... carry on! :sweat_smile:

**Pull request merged**